### PR TITLE
Video Listing

### PR DESCRIPTION
**Pull request by vijaytembugade**
- Added Video Listing Page
- Video Context
- Styling

### CODE REVIEW

File: src/Componets/VideoCard/VideoCard.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./VideoCard.css";
+
+const VideoCard = ({ video }) => {
```
Comment (sanathkumar26700) [inline src/Componets/VideoCard/VideoCard.js:4]: Suggestion : Could destructure video one level further to avoid repetition in code.

**Review by sanathkumar26700 — APPROVED** [review]
✔Clean Code
✔Clean UI
✔ Everything works fine

File: src/Componets/Header/Header.js [hunk 2]
```diff
@@ -27,12 +27,15 @@ const Header = () => {
...
-          class="material-icons md-36"
+          className="material-icons md-36"
           onClick={() => setShowDropDownMenu(!showDropDownMenu)}
         >
           expand_circle_down
         </span>
-        <span>
+        <span
+          onMouseEnter={() => setShowDropDownMenu(true)}
+          onMouseLeave={() => setShowDropDownMenu(false)}
```
Comment (MiheerTamkhane) [inline src/Componets/Header/Header.js:37]: Very Cleaver approach.

**Review by MiheerTamkhane — APPROVED** [review]
Code is good and readable UI is looking nice.
lot of code is adding data in db, nothing more by my side goot to go.

**Pull request merged**